tensorflow/compiler/xla/service/gpu/miopen_conv_algorithm_picker.cc

```diff
@@ -77,6 +76,65 @@ StatusOr<se::DeviceMemory<uint8>> ScratchAllocator::AllocateBytes(
   return se::DeviceMemory<uint8>(buffer_addr);
 }
 
+StatusOr<se::dnn::ConvolutionKind> GetDnnConvolutionKind(
+    const HloCustomCallInstruction* conv) {
+  TF_ASSIGN_OR_RETURN(CudnnConvKind kind, GetCudnnConvKind(conv));
```

Comment:
A better place for this function is probably not here, but in the place of [`GetCudnnConvKind`](https://github.com/tensorflow/tensorflow/blob/3ead54282c2828990794c1f266e0482b17a5521c/tensorflow/compiler/xla/service/gpu/ir_emission_utils.cc). Realized this when addressing review feedback from [#30884](https://github.com/tensorflow/tensorflow/pull/30884)